### ISSUE TITLE
Implement friendship methods from tweepy

### ISSUE THREAD

**Issue by wdm0006** [wdm0006/twitter-pandas#4]
Implement twitterpandas methods for each of the dataset returning methods in tweepy's friendship methods grouping.

Examples of how to go about this can be found in the existing client.py, where the user methods group has been implemented. Use cursors wherever possible.

**Comment by kevakil** [comment]
I'll try it.

**Comment by wdm0006** [comment]
Awesome, let me know if you have any questions of if I can otherwise help.

**Comment by kevakil** [comment]
Just to make sure, I should not implement create_friendship and destroy_friendship, yeah?

**Comment by wdm0006** [comment]
Correct

**Comment by kevakil** [comment]
I'm having a tough time understanding how the input should be for the exists_friendship method and, as a result, how to implement the method. The tweepy API reference says the input for exists_friendship should be:
    user_a – The ID or screen_name of the subject user.
    user_b – The ID or screen_name of the user to test for following.

However, I was thinking of using my implementation of show_friendship as a framework, but that would require the following input:
    source_id – The user_id of the subject user.
    source_screen_name – The screen_name of the subject user.
    target_id – The user_id of the target user.
    target_screen_name – The screen_name of the target user.

Should I be using show_friendship as a framework for this method? If so, how should I format the input? If not, how should I go about implementing exists_friendship?

**Comment by wdm0006** [comment]
That's a good question.  At this stage, I think we should prioritize API consistency over consistency with tweepy.   Our goal is to provide an intuitive interface to twitter data via pandas, not to copy the tweepy API (despite the first batch of work being to pretty much just copy the tweepy API).  

So if you have an opportunity to make the values passed to two similar functions the same instead of slightly different, that's great.

Long story short, I think using your previous method as a framework is a good idea.
